context/app/api/vitessce.py

```diff
@@ -421,7 +454,7 @@ def _build_image_schema(self, image_rel_path):
         """
 
         schema = {}
-        schema["name"] = Path(image_rel_path).name
+        schema["name"] = name if name else Path(image_rel_path).name
```

Comment:
```suggestion
        schema["name"] = name if name is not None else Path(image_rel_path).name
```
Let's be more explicit, rather than relying on the falsiness of `""`.